caikit/config/config.yml

```diff
@@ -123,8 +126,6 @@ runtime:
     grpc:
         enabled: true
         port: 8085
-        # Number of workers with which we will run the gRPC server
-        server_thread_pool_size: 5
```

Comment:
I think we need to keep this in so that env var overrides work. We can default it to `null` or `0` though.

Reply:
🌶️